app/crawler/mbc_api.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any
+import aiohttp
+import json
+from datetime import date
+from app.crawler.base import BaseNewsCrawler, Article
+from app.crawler.utils.json_cleaner import sanitize_js_style_json
+from common.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+class MbcNewsApiCrawler(BaseNewsCrawler):
+    async def fetch_articles(self) -> list[Article]:
+        logger.info("[MBC API] fetch_articles 시작")
+        current_id, data_id = await self._get_today_ids()
+        return await self._fetch_news(current_id, data_id)
+
+    async def _get_today_ids(self) -> tuple[str, str]:
+        cal_url = "https://imnews.imbc.com/news/2025/politics/cal_data.js"
+        logger.info(f"[MBC API] cal_data.js 요청: {cal_url}")
```

Comment:
```suggestion
        logger.info("[MBC API] cal_data.js 요청: %s", cal_url)
```

logging 모듈을 사용하는김에 실제로 값이 사용되는경우 평가(eval)가 진행되는 특성을 활용해보시죠

각 로그는 info, debug, warning, exception 등의 레벨을 가지고있는데요 실행되는 런티임의 로그 레벨에 따라 아얘 노출되지 못하는 경우가 종종 생깁니다.

예를들면 `logger.debug(...)` 의 경우 DEBUG 모드가 아닌경우 출력되지 않는데요
logging 모듈이 내부적으로 tuple 형식으로 인자를 받고, 로그 메시지를 실제로 출력해야하는 상황에서만 포매팅을 진행하기 때문입니다.

이를 흔히 `lazy evaluation` 이라 부릅니다.

지금 사용하신 방식과 같은 f-string 문자열을 사용하게되면 로그가 출력되지 않는데도 불구하고 연산이 발생하기 때문에 프로그램 규모가 커질 수록 불리하다는 단점이 생깁니다.